lib/praxis/middleware_app.rb

```diff
@@ -0,0 +1,30 @@
+module Praxis
+  class MiddlewareApp
+
+    attr_reader :target
+
+    # Initialize the application instance with the desired args, and return the wrapping class.
+    def self.for( **args )
+      Praxis::Application.instance.setup(args)
```

Comment:
technically `**args`. but there's only 1 param explicitly defined in the signature, so i believe they both are the same in this case.

Will change it though.